features/messages/impl/src/test/kotlin/io/element/android/features/messages/voicemessages/composer/VoiceMessageComposerPresenterTest.kt

```diff
@@ -323,6 +340,7 @@ class VoiceMessageComposerPresenterTest {
             awaitItem().eventSink(VoiceMessageComposerEvents.PlayerEvent(VoiceMessagePlayerEvent.Play))
             awaitItem().eventSink(VoiceMessageComposerEvents.SendVoiceMessage)
             assertThat(awaitItem().voiceMessageState).isEqualTo(aPlayingState().toSendingState())
+            skipItems(1) // Duplicate sending state
```

Comment:
I needed to add this after [this change](https://github.com/vector-im/element-x-android/pull/1758/files#diff-3f2fbb85a53da92f3e329901ee9c3e3710c53c453b1d816ca18fc02f28e88217R57) but couldn't get to the bottom of why. The `state` flow still emits the exact same values in this test.